templates/zammad/0/docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+version: '2'
+
+services:
+
+  elasticsearch:
+    image: zammad/zammad-docker-compose:elasticsearch
+    labels:
+      io.rancher.container.pull_image: always
+    restart: always
+
+  nginx:
+    depends_on:
+      - zammad
+    image: zammad/zammad-docker-compose:nginx
+    labels:
+      io.rancher.container.pull_image: always
+    links:
+      - zammad-railsserver
+      - zammad-websocket
+    ports:
+      - "80:80"
+    restart: always
+    volumes:
+      - data-zammad:/home/zammad
+
+  postgresql:
+    image: zammad/zammad-docker-compose:postgresql
+    labels:
+      io.rancher.container.pull_image: always
+    restart: always
+
+  zammad:
+    image: zammad/zammad-docker-compose:zammad
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+    restart: on-failure
+    volumes:
+      - data-zammad:/home/zammad
+
+  zammad-railsserver:
+    image: zammad/zammad-docker-compose:zammad-railsserver
+    labels:
+      io.rancher.container.pull_image: always
+    links:
+      - elasticsearch
+      - postgresql
+      - zammad
+    depends_on:
+      - elasticsearch
+      - postgresql
+      - zammad
+    restart: always
+
+  zammad-scheduler:
+    image: zammad/zammad-docker-compose:zammad-scheduler
+    labels:
+      io.rancher.container.pull_image: always
+    links:
```

Comment:
# Set link based in elasticsearch service
--      - elasticsearch
++      - ${zammad_elasticsearch_service}: elasticsearch